lib/compat/wordpress-6.5/interactivity-api/interactivity-api.php

```diff
@@ -58,16 +58,20 @@ function wp_interactivity_process_directives_of_interactive_blocks( array $parse
 				};
 
 				/*
-				* Uses a priority of 20 to ensure that other filters can add additional
-				* directives before the processing starts.
-				*/
-				add_filter( 'render_block_' . $block_name, $process_interactive_blocks, 20, 2 );
+				 * Uses a priority of 100 to ensure that other filters can add additional
+				 * directives before the processing starts.
+				 */
+				add_filter( 'render_block_' . $block_name, $process_interactive_blocks, 100, 2 );
 			}
 		}
 
 		return $parsed_block;
 	}
-	add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks' );
+	/*
+	 * Uses a priority of 100 to ensure that other filters can edit $parsed_block
+	 * without crashing the SSR.
+	 */
+	add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks', 100 );
```

Comment:
[core has 1 as the last argument here](https://github.com/WordPress/wordpress-develop/blob/0c24793d257cc4ce2f929a6e911f6b94bf97f19a/src/wp-includes/interactivity-api/interactivity-api.php#L69-L73), although I think that's the default and maybe it should be omitted in core.

Reply:
True

Reply:
Yes it should be omitted, pointless to add the default